lsp/tsconfig.packages.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
The monorepo root has its own dedicated tsconfig.json. The name of this file is intended to represent that it is the base tsconfig for other packages in the repo.

The properties omitted from this file (like rootDir and outDir) are deliberate -- when they were moved to this file, they are interpreted as if they are from this file/location. So you'd have a project trying to compile code from the root's /src folder instead of the package's /src folder.